src/Elmish/HTML/Events/Methods.js

```diff
@@ -0,0 +1,4 @@
+export const getModifierState_ = key => e => (e.getModifierState && e.getModifierState(key)) || false
+
+export const preventDefault_ = e => () => e.preventDefault && e.preventDefault()
+export const stopPropagation_ = e => () => e.stopPropagation && e.stopPropagation()
```

Comment:
Are we re-implementing this because we have different types than `Web.Event.Event` and/or want to have fewer dependencies? https://github.com/purescript-web/purescript-web-events/blob/c8a50893f04f54e2a59be7f885d25caef3589c57/src/Web/Event/Event.js#L39

Reply:
Yes, different type. I guess I could have `unsafeCoerce`d instead. This felt cleaner.

On the other hand, one might argue that `Web.Events.Event.preventDefault` is not actually the same thing, because its argument must be a _browser_ event, not a React event. The two are different things, but the function works anyway due to duck typing.